apps/explorer/src/app/components/txs/details/chain-events/tx-builtin-deposit.tsx

```diff
@@ -0,0 +1,46 @@
+import { t } from '@vegaprotocol/react-helpers';
+import { TableRow, TableCell } from '../../../table';
+import type { components } from '../../../../../types/explorer';
+import { AssetLink, PartyLink } from '../../../links';
+
+interface TxDetailsChainEventBuiltinDepositProps {
+  deposit: components['schemas']['vegaBuiltinAssetDeposit'];
+}
+
+/**
+ * Someone deposited some of a builtin asset. Builtin assets
+ * have no value outside the Vega chain and should appear only
+ * on Test networks.
+ */
+export const TxDetailsChainEventBuiltinDeposit = ({
+  deposit,
+}: TxDetailsChainEventBuiltinDepositProps) => {
+  if (!deposit) {
+    return <>{t('Awaiting Block Explorer transaction details')}</>;
```

Comment:
Chore (non-blocking): it's a tiddler, but this message appears 18 times in the PR, I think it's worth making into a shared component